src/java_instance_wrapper.cpp

```diff
@@ -4,7 +4,7 @@
 Map<String, JavaInstanceWrapper::ClassCache> JavaInstanceWrapper::CLASS_CACHE = Map<String, ClassCache>();
 
 JavaInstanceWrapper::JavaInstanceWrapper(const char* p_class_name, jni::JObject p_wrapped,
-                                         jni::JObject p_class_loader) : class_name(p_class_name) {
+                                         jni::JObject p_class_loader) : classname(p_class_name) {
```

Comment:
why change the field name? It doesn't follow the convention.

Reply:
it is because `KtPropertyInfo` has a variable class_name. As it inherits from JavaInstanceWrapper, they should have differents name. I forgot to set a better one. Also I don't have strong opinion about changing name of `JavaInstanceWrapper` member or `KtPropertyInfo`.

Reply:
Ahhh, didn't realize that. Maybe you can rename the one in `JavaInstanceWrapper` to `java_class_name`?

Reply:
yep that's sound better